bin/run-pacta

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env bash
```

Comment:
When working in Docker, I prefer that scripts are `sh` compliant, rather than assuming that `bash` is available. in particular, `alpine` doesn't  have `bash` (instead it has `ash`, IIRC)